frontend/src/components/ArticleCard.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode, useState } from 'react'
+import ArticleIcon from '../assets/article.svg'
+
+
+const ArticleCard = ({ title, body }: { title: string, body: string }) => {
+  return (
+    <div className="card w-11/12 bg-base-100 shadow-xl">
+      <div className="card-body justify-center">
+        <ArticleIcon></ArticleIcon>
```

Comment:
You should only be returning a <ArticleIcon /> here. See PR made by @DylanTom and @hahnandrew if you need help.

Reply:
are you referring specifically to <ArticleIcon>?